### DISCUSSION TITLE
Something like np.outer

### DISCUSSION THREAD

**Discussion by tolios** [arogozhnikov/einops#280] [Q&A] [answered]
Hey there, I was wondering if there exists a feature, or a way to implement a simple outer operation like the one offered by numpy: https://numpy.org/doc/stable/reference/generated/numpy.outer.html. It might be straightforward already, but if not I would find it nice to have it as a feature. Like: einouter(a, b, "i, j -> i j").

Sorry again if this is already possible, i just found out this library.

**Comment by arogozhnikov** [comment, marked as answer]
yes, that's possible:

```python
einops.einsum(a, b, 'i, j -> i j')
```
